invoicing_app/management/commands/generate_tax_receipts.py

```diff
@@ -166,24 +167,11 @@ def handle(self, **options):
 
         if options['event_id']:
             self.event_id = options['event_id']
-            self.query = self.query.replace(
-                'CONDITION_MASK',
-                'AND `Events`.`id` = ' + str(self.event_id)
-            )
+            self.conditional_mask = 'AND `Events`.`id` = {}'.format(self.event_id)
```

Comment:
nit: Not sure if it's possible in this particular context, but when working with queries, for safety measures it's preferred to insert values (like event_id and user_id) using the cursor and not using format, because the cursor has some special checks that help preventing some attacks.
It's not that worrying in this particular case as this should only be run by Eventbrite admins, but if you can apply it as a good practice it would be nice.

Reply:
Ok! Thanks!